.github/workflows/stale.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# This workflow alerts and then closes the stale issues/PRs after specific time
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+
+name: 'Close stale issues and PRs'
+"on":
+  schedule:
+  - cron: "30 1 * * *"
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: 'actions/stale@v7'
+      with:
+        # Comma separated list of labels that can be assigned to issues to exclude them from being marked as stale.
+        exempt-issue-labels: 'override-stale'
+        # Comma separated list of labels that can be assigned to PRs to exclude them from being marked as stale.
+        exempt-pr-labels: "override-stale"
+        # Limit the No. of API calls in one run default value is 30.
+        operations-per-run: 500
```

Comment:
Why has the default limit been increased to 500?

Reply:
It will pick 3-4 issues/PRs only with default value 30 to stale/close. Increasing it to 500 so it will iterate over all the issues for stale/close.We have per hour API limit 5000/per workflow.